rviz_default_plugins/src/rviz_default_plugins/displays/marker/markers/triangle_list_marker.cpp

```diff
@@ -30,18 +30,25 @@
 
 #include "rviz_default_plugins/displays/marker/markers/triangle_list_marker.hpp"
 
+#include <iostream>
+#include <fstream>
```

Comment:
Do we need to include these headers?

Reply:
Nope! Residuals from debugging/other attempts at solutions!